xml/System.IO/DirectoryInfo.xml

```diff
@@ -3867,8 +3867,6 @@ The following example refers to the parent directory of a specified directory.
       <AssemblyInfo>
         <AssemblyName>System.Runtime</AssemblyName>
         <AssemblyVersion>6.0.0.0</AssemblyVersion>
-        <AssemblyVersion>7.0.0.0</AssemblyVersion>
-        <AssemblyVersion>8.0.0.0</AssemblyVersion>
```

Comment:
I would rather do nothing here. The docs should align with the ref assembly shape and doing this would break that. If we backported https://github.com/dotnet/runtime/pull/91354 to 7 and 8 it would make sense to make this change, but I don't think is a change worth backporting though. 

@carlossanlop @bartonjs thoughts?